CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - `Improbable.Gdk.EntityTemplate` is now mutable and exposes a set of APIs to add, remove, and replace component snapshots
     - This replaces the `Improbable.Gdk.Core.EntityBuilder` class.
     - These changes also allow you to reuse an `EntityTemplate` more than once.
+- Fixed Memory crash in TransformSynchronization module.
```

Comment:
Nit: capital M in memory.

Also maybe a better message is:

`Fixed a crash in the TransformSynchronization module where an integer overflow would cause a memory crash`

Reply:
`Fixed an issue in the TransformSynchronization module where an integer overflow would cause a memory crash`

Reply:
Wot he said ^^^